get_started/what_is_citus.rst

```diff
@@ -14,7 +14,8 @@ I/O bandwidth can lead to significant performance improvements for multi-tenant
 SaaS applications, customer-facing real-time analytics dashboards, and time
 series workloads.
 
-**Two Ways to Get Citus:**
+Two Ways to Get Citus:
+$$$$$$$$$$$$$$$$$$$$$$
```

Comment:
```
reading sources... [100%] use_cases/timeseries


Warning, treated as error:
/home/awolk/citus_docs/get_started/what_is_citus.rst:64:Title level inconsistent:

Multi-Tenant SaaS Database
--------------------------
make: *** [Makefile:4: docs] Error 2
(.venv) awolk@DESKTOP-J1E0E13:~/citus_docs$
```

Fails to build with this change. Please see Sphinx documentation on Headings - https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Why $ here? What leveling of headings are you aiming for in boths files?

Reply:
Trying to do h4. The reference_sql.rst had $$$ for an existing h4 on the page.

Reply:
I tested a bit what passes through the build.

If I do:
```rst
lvl 1
====

lvl 2
----

lvl 3
^^^

lvl 4
$$$$
```

it builds.

```rst
lvl 1
====

lvl 2
----


lvl 4
$$$$
```

this also builds but is rendered as H3.

Any variant where lvl 2 is skipped fails.

Hence, these are not assigned to specific H1,2,3,4 levels and will render differently depending if previous levels are present. You will need to test with local builds to make sure this renders as you expect.

Reply:
I see. I think set as lvl 3 for the what_is_citus page.

Reply:
Accessibility fixes in this branch:
- change bold lines to headers for easy keyboard navigation (rst edits)
- fix contrast issues for links (css edits)